index.html

```diff
@@ -1,39 +1,348 @@
-    <title>Document</title>
-</head>
-<body>
-
-    
-        <nav class="nav">
-            <label class="logo">KAZANEZA GENTIL</label>
-            <a href="#">
-                <img class="navImage" src="./Images/ic_menu.svg" alt="Navigation menu">
-            </a>
-        </nav>
-    
-        <header class="container">
-            <div class="headline"> Hey there. <br/> I'm Kazaneza</div>
-            <div class="title">I’am a Software Developer</div>
-            <div class="introduction"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, on’t hestiate to contact me.</div>
-
-            <div class="icons">
-                <div class="button twitter"><img  src="./images/Social Media.png" alt=""></div>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/mobile.css" />
+    <link
+      media="screen and (min-width: 768px)"
+      rel="stylesheet"
+      href="./styles/desktop.css"
+    />
+    <link
+      href="http://fonts.googleapis.com/css?family=Roboto"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <title>Portfolio Kazaneza</title>
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <label class="logo"><a href="">KAZANEZA GENTIL</a></label>
+        <ul class="desktop-nav">
+          <li class="desktop-link"><a href="#">Portfolio</a></li>
+          <li class="desktop-link"><a href="#">About</a></li>
+          <li class="desktop-link"><a href="#">contact</a></li>
+        </ul>
+        <a href="#" class="mobile-nav">
+          <img
+            class="navImage"
+            src="./Images/ic_menu.svg"
+            alt="Navigation menu"
+          />
+        </a>
+      </nav>
+    </header>
+
+    <section class="container">
+      <div class="proflle-content">
+        <h1 class="headline">
+          Hey there. <br />
+          I'm Kazaneza
+        </h1>
+        <h2 class="title">I’am a Software Developer</h2>
+        <p class="introduction">
+          I can help you build a product, feature or website Look through some
+          <br />
+          of my work and experience! If you like what you see and have a
+          project<br />
+          you need coded, on’t hestiate to contact me.<br />
+        </p>
+      </div>
+
+      <div class="icons">
+        <ul>
+          <li>
+            <a href=""><img src="./images/Group-1.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/Group-2.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/Vector-1.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/Vector.png" alt="" /></a>
+          </li>
+        </ul>
```

Comment:
- [ ] Kindly make sure the `space` between these elements match with the `space` in the Figma design.

Your Design
![Screenshot from 2022-12-10 18-12-50](https://user-images.githubusercontent.com/80458199/206865138-08f02595-fb50-45da-bd07-5921e7d7de90.png)

Figma Design

![Screenshot from 2022-12-10 18-27-56](https://user-images.githubusercontent.com/80458199/206865155-fbe658ff-f1a3-495a-8504-c7aedbc2db8b.png)

Reply:
Fixed.